.github/pull_request_template.md

```diff
@@ -13,3 +13,19 @@ fixes # <!-- Creating an issue before an Pull Request never hurts -->
 ## How Has This Been Tested?
 
 ## Screenshots (if appropriate):
+=======
```

Comment:
what's this?

Reply:
It was in the previous repo of yours. Screenshot if can be added!
I am adding basic PR template and for others as well!